src/cascadia/TerminalSettingsModel/Resources/en-US/Resources.resw

```diff
@@ -445,7 +445,7 @@
     <value>Switch to the last tab</value>
   </data>
   <data name="TabSearchCommandKey" xml:space="preserve">
-    <value>Search for tab</value>
+    <value>Search for tab...</value>

```

Comment:
Should this (these?) use … instead?

Reply:
No, I mean whether we should use the dedicated ellipsis Unicode character for this: https://www.compart.com/en/unicode/U+2026

Reply:
Oh! My stance is "eh". Is there a value to switching over to the unicode character? Screen readers seem to handle it no differently.